<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_Parameters">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>370</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_StartTime"/>
        </item>
        <item row="10" column="1">
         <widget class="QRadioButton" name="radioButton_AllArrays">
          <property name="text">
           <string>All</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QRadioButton" name="radioButton_FirstArray">
          <property name="text">
           <string>1st</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QRadioButton" name="radioButton_OTB_csv">
          <property name="text">
           <string>OTB (.csv)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_EndTime"/>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_NoiseEnd">
          <property name="text">
           <string>     Noise end (s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Browse">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLineEdit" name="lineEdit_NoiseEnd"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_SelectingBadChannels">
          <property name="text">
           <string>Selecting bad channels:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_SamplingFreq">
          <property name="text">
           <string>     Sampling freq.</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_SNRThreshold">
          <property name="text">
           <string>     SNR threshold</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_EMGFile"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_EndTime">
          <property name="text">
           <string>     End (s)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_StartTime">
          <property name="text">
           <string>     Start (s)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_SamplingFreq"/>
        </item>
        <item row="16" column="1">
         <widget class="QPushButton" name="pushButton_ShowBadChannels">
          <property name="text">
           <string>Show bad channels</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLineEdit" name="lineEdit_NoiseStart"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_EMGFile">
          <property name="text">
           <string>EMG file</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QRadioButton" name="radioButton_SecondArray">
          <property name="text">
           <string>2nd</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_NoiseStart">
          <property name="text">
           <string>     Noise start (s)</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLineEdit" name="lineEdit_SNRThreshold"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_EndTime_2">
          <property name="text">
           <string>     Selected array:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="radioButton_GUI_Muovi_csv">
          <property name="text">
           <string>GUI_Muovi (.csv)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>271</width>
         <height>354</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="0">
         <widget class="QLabel" name="label_lowcutFreq">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>     lowcut freq.</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QCheckBox" name="checkBox_SIL">
          <property name="text">
           <string>SIL</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_MaxRefIter">
          <property name="text">
           <string>Max. refinement iter.</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="lineEdit_SILThreshold">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_order">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_MinDistance"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_MinDistance">
          <property name="text">
           <string>Min. distance between peaks</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_highcutFreq">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_ExtFactor"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_Iterations"/>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="lineEdit_MaxSepIter"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_DiscardChannels">
          <property name="text">
           <string>Discard channels</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_Iterations">
          <property name="text">
           <string>Iterations</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_SILThreshold">
          <property name="text">
           <string>     threshold</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLineEdit" name="lineEdit_xTolerance"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_DiscardChannels"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_order">
          <property name="text">
           <string>     order</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLineEdit" name="lineEdit_MaxRefIter"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_xTolerance">
          <property name="text">
           <string>     x_tolerance</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_BandpassFilter">
          <property name="text">
           <string>Bandpass filter</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_ExtFactor">
          <property name="text">
           <string>Extension factor</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_MaxSepIter">
          <property name="text">
           <string>Max. separation iter.</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_highcutFreq">
          <property name="text">
           <string>     highcut freq.</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_lowcutFreq">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_SaveAs">
          <property name="text">
           <string>Save as</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_SaveAs"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_RealtimeDecomposition">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>440</y>
         <width>226</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Realtime Decomposition</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Decompose">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>410</y>
         <width>226</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Decompose</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
